jetty-util/src/main/java/org/eclipse/jetty/util/component/FileNoticeLifeCycleListener.java

```diff
@@ -22,7 +22,9 @@
 /**
  * A LifeCycle Listener that writes state changes to a file.
  * <p>This can be used with the jetty.sh script to wait for successful startup.
+ * @deprecated use {@code org.eclipse.jetty.server.StateLifeCycleListener} instead
  */
+@Deprecated
 public class FileNoticeLifeCycleListener implements LifeCycle.Listener
```

Comment:
Can you make it extend the new `StateLifeCycleListener` rather than duplicating?

Reply:
Wrong way for the dependency.
The jetty-server StateLifeCycleListener cannot be referenced in jetty-util